pixiedust/utils/sampleData.py

```diff
@@ -153,12 +159,19 @@ def getType(t):
 
         res = open(path, 'r').read()
 
+        # Force load to pandas
+        if Environment.hasSpark and self.forcePandas:
```

Comment:
rather than duplicate the pandas load/read could we possibly do something like
```
if Environment.hasSpark and not self.forcePandas:
    if Environment.sparkVersion == 1:
        # read/load json using spark1
    else:
        # read/load json using spark2
else:
    # read/load json using pandas
```
otherwise, all looks good